.github/workflows/ci.yml

```
@@ -37,7 +37,7 @@ jobs:
           # avoid building unrelated products for testing by specifying the test product explicitly
           swift build --product TokamakPackageTests
           `xcrun --find xctest` .build/debug/TokamakPackageTests.xctest ||
-            (cp -r `find /var/folders -iname RenderingTests` . ; exit 1)
+            (cp -r /var/folders/*/*/*/*Tests . ; exit 1)
```

Double star should be able to traverse all nested directories and is hopefully more reliable.

```suggestion
            (cp -r /var/folders/**/*Tests . ; exit 1)
```

---

That didn't work. Going back to the single wildcard does work, though these is a bit extra in there. Images look identical to me.